app/jobs/indexer_job.rb

```diff
@@ -0,0 +1,24 @@
+class IndexerJob
+  include Sidekiq::Worker
+  sidekiq_options queue: 'elasticsearch', retry: false
```

Comment:
I guess we wouldn't be editing the records that much, so we should actually probably set `retry: true` to ensure they would be indexed.

The usual use case I'm guessing is a lot more updates so if we miss one, hey, no big deal.

Also, I was considering something like a catch-all regular nightly task to just re-index all the resources, regardless. Wouldn't be a huge deal and would ensure everything is up to date. Dunno, just spitballin'.

I reverted this to true in dd9fa6c.